lib/order.rb

```diff
@@ -0,0 +1,36 @@
+class Order
+  attr_reader :menu, :dishes, :quantity, :total
+
+  def initialize(menu_klass)
+    @menu = menu_klass
+    @dishes = []
+    @quantity = 0
+  end
+
+  def choose(dish, quantity=1)
+    fail "This is not on the menu" unless self.has_dish?(dish)
+    quantity.times do
+      menu.select { |k,v| dishes.push([k, v]) if k == dish }
+    end
+  end
+
+  def calculate_quantities(order)
+    b = Hash.new(0)
+    dishes.each { |v| b[v] += 1 }
+    b.each { |k, v| puts "#{v} x #{k[0]}" }
+    quantity = dishes.length
+  end
+
+  def calculate_cost(order)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `order`. If it's necessary, use `_` or `_order` as an argument name to indicate that it won't be used. You can also write as `calculate_cost(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)